### ISSUE TITLE
$variable not being resolved to its value

### ISSUE THREAD

**Issue by brumm** [sass/node-sass#860]
environment is a ember-cli app

```sh
$ node_modules/broccoli-sass/node_modules/node-sass/bin/node-sass -v
node-sass	3.0.0-beta.5	(Wrapper)	[JavaScript]
libsass  	3.2.0-beta.5	(Sass Compiler)	[C/C++]
```

i'm using i'm using http://bourbon.io/docs/#font-face to include some fonts:

```scss
@include font-face(MavenPro, 'fonts/maven-pro/regular/maven-pro-regular-webfont', 100);
```

the rendered css output is:
```css
@font-face {
  font-family: MavenPro;
  font-style: normal;
  font-weight: 100;
  src: url($file-path) format("embedded-opentype"), url($file-path) format("woff2"), url($file-path) format("woff"), url($file-path) format("truetype"), url($file-path) format("svg"); }
```

note that `$file-path` is being rendered out, and not correctly resolved to its value.

the relevant mixin code is here: https://github.com/thoughtbot/bourbon/blob/master/app/assets/stylesheets/helpers/_font-source-declaration.scss#L37

changing line line 37 to
```scss
$src: append($src, url("#{$file-path}") $font-format, comma);
```

fixes this and outputs the correct value.
i tried to replicate the behavior in sassmeister, but it works as expected (the libsass version doesnt match though):
http://sassmeister.com/gist/5b76b269b32cca33dbeb

**Comment by saper** [comment]
Seems similar to #850 to me

**Comment by saper** [comment]
This is a libsass issue sass/libsass/issues#1087 . A proper fix is on the way, but some simple fix has been already included in the `libsass` master.

**Comment by xzyfer** [comment]
This fixed in Libsass and will be in the next beta.

**Comment by xzyfer** [comment]
This has been resolved in 3.0.0-beta.7.

```
npm install node-sass@beta
```
